zetaclient/bitcoin_client.go

```diff
@@ -73,9 +73,11 @@ type BitcoinChainClient struct {
 }
 
 const (
-	minConfirmations = 0
-	maxHeightDiff    = 10000
-	btcBlocksPerDay  = 144
+	minConfirmations          = 0
+	maxHeightDiff             = 10000
+	btcBlocksPerDay           = 144
+	bigValueSats              = 200000000
+	bigValueConfirmationCount = 6
```

Comment:
Can we add comments, what these values represent and what are they used for?

Reply:
[added comments](https://github.com/zeta-chain/node/pull/1674/commits/fd01652a846366c4b5f6e898e113f6e5f43695ac)